fs/mount.c

```diff
@@ -23,6 +23,21 @@ void fs_register(const struct fs_ops *fs) {
     assert(!"reached filesystem limit");
 }
 
+char * get_filesystems(void) {
+    char *fs_list = calloc(MAX_FILESYSTEMS * 50, sizeof(char)); // Reasonable assumption?
+    unsigned int i;
+
+    for ( i = 0; i < MAX_FILESYSTEMS; i++ ) {
```

Comment:
nit: spacing, `for (i = 0; i < MAX_FILESYSTEMS; i++) {`

Reply:
Too much shell script programming?  I'm a fan of white space but that is excessive.  Noted.